src/components/cookie-banner/index.md.njk

```diff
@@ -29,95 +40,112 @@ Check with your organisation's privacy expert to see how data protection legisla
-- remove the cookie banner if the user clicks on the ‘Hide this message’ button
-- prevent the cookie banner being shown when the user moves on to a new page
+Include the name of the service in the banner heading to help users understand that the cookies you’re talking about are different from the ones set by the main GOV.UK platform.
 
-## If the user’s device is not running JavaScript
+You’ll need to change the example cookie banner text if your service:
 
-If the user’s device is not running JavaScript, display a version of the cookie banner which tells them what to do in order to set or change their cookie preferences. Do not set any non-essential cookies, because the user has not consented to them.
+- allows third parties to set cookies (tell the user that both your organisation and other organisations will be setting cookies)
+- uses cookies for reasons other than collecting analytics information or remembering the user’s settings
 
-{{ example({group: "components", item: "cookie-banner", example: "no-js", html: true, nunjucks: true, open: false}) }}
+Keep the text as short as possible while making sure it’s an accurate description of how you use cookies. For example, if you use more than one ‘functional’ cookie and there’s not enough space to mention what each of them does, you could ask for permission to set cookies so ‘you can use as many of the service’s features as possible’.
 
-If your service only uses essential cookies, display a different version of the cookie banner to users whose devices are not running JavaScript.
+[See the cookies page for more advice on writing about cookies(/patterns/cookies-page/)].
 
-{{ example({group: "components", item: "cookie-banner", example: "essential-no-js", html: true, nunjucks: true, open: false}) }}
+### If you’re using essential cookies and analytics cookies
 
-## Getting consent from users who are not running JavaScript
+You can use this example text for a service which sets essential and analytics cookies - set by your organisation to collect analytics information about how well the service is performing.
 
-[Some users will not be running JavaScript on their devices](https://gds.blog.gov.uk/2013/10/21/how-many-people-are-missing-out-on-javascript-enhancement/). If you’re using non-essential cookies and want to get consent from these users, you’ll need an approach that does not depend on JavaScript.
+{{ example({group: "components", item: "cookie-banner", example: "default", html: true, nunjucks: true, open: false}) }}
 
-Show the replacement message confirming that the user has accepted or rejected cookies when they submit the form and the page has reloaded.
+### If you’re using more than one type of non-essential cookie
 
-Here’s an example showing how it should work.
+You can use this example text for a service that set:
+- essential cookies
+- analytics’ cookies
```

Comment:
```suggestion
- analytics cookies
```
Not sure this needs an apostrophe? Haven't noticed it used like this elsewhere but might be worth a check